site/docs/reference/connectors-datacatalog/README.md

```diff
@@ -28,6 +28,10 @@ Class | Method | HTTP request | Description
  - [ResourceMetadata](Models/ResourceMetadata.md)
  - [UpdateAssetRequest](Models/UpdateAssetRequest.md)
  - [UpdateAssetResponse](Models/UpdateAssetResponse.md)
+ - [db2](Models/db2.md)
+ - [fybrik-arrow-flight](Models/fybrik-arrow-flight.md)
```

Comment:
See previous comment about "fybrik"-arrow-flight and whether there is anything specific to fybrik in this connection type